src/components/standup/index.tsx

```diff
@@ -83,12 +87,12 @@ const StandUpContainer: FC = () => {
                         <form
                             className={styles.standupForm}
                             onSubmit={handleFormSubmission}
+                            role="form"
```

Comment:
I think this role is automatically defined by accessibility roles in testing, I don't think we need to add this role over here.

Reply:
yes we could do it but I am getting below error while removing it so had to put role into it
sharing the error screenshot below
![image](https://github.com/Real-Dev-Squad/website-status/assets/82165483/6c9118bf-ab46-4ece-ad0a-092a5ff01581)

Reply:
then why isn't it able to access it with a role> something might be off, could find out and fix it please?

Reply:
fixed